cpp/include/cudf/stream_compaction.hpp

```diff
@@ -18,6 +18,17 @@
 #define STREAM_COMPACTION_HPP
 
 #include "cudf.h"
+#include "types.hpp"
+
+
+/**
+ * @brief Choices for drop_duplicates API for retainment of duplicate rows
+ */
+enum duplicate_keep_option {
```

Comment:
This should go inside the `cudf` namespace.

Reply:
moved.